astropy/modeling/core.py

```diff
@@ -1431,9 +1431,9 @@ def inverse(self):
     def inverse(self, value):
         if not isinstance(value, (Model, type(None))):
             raise ValueError(
-                "The ``inverse`` attribute may be assigned a `Model` "
-                "instance or `None` (where `None` explicitly forces the "
-                "model to have no inverse."
+                "The ``inverse`` attribute may be assigned a "
+                "`~astropy.modeling.Model` instance or `None` (where "
```

Comment:
I don't think you have to change it in an error message? In fact I would argue the backticks are also unnecessary.

Reply:
Yeah, I was wondering why a reference was included in an error message.  I'll just remove it.

Reply:
Fixed, thanks!